### ISSUE TITLE
Add dictionary

### ISSUE THREAD

**Issue by LAfricain** [postiffm/bibledit-desktop#15]
This is not really an issues, more a question (but I don't know how to ask it on github).

How to add a dictionary and how bibledit-desktop use the dictionaries of the system? Because in the list of the dictionaries I have the fr and the en dic. Maybe it is the language installed on the OS?

**Comment by postiffm** [comment]
This is a mystery to me right now. Bibledit-desktop uses enchant library as an interface to handle spelling. The libraries for enchant reside in special directories. On my Linux Ubuntu 16 system, I open the Dictionaries window and it shows a bunch of language options in English and in French--presumably because I've installed the French localization package (but I forgot what that is). I can pick one of those dictionaries and it works fine.

However, on Windows, the list of dictionaries is empty except for the standard "shared dictionary" (per project) and "global dictionary" (per user). I cannot figure out where to put dictionary files to get the Windows version to recognize them--for English, for example, or French. I'm sure once I find the right place to put them, all will be well.

**Comment by postiffm** [comment]
Same result if run bibledit-gtk.exe from the mingw64 shell--no spelling dictionaries are shown despite the fact that they are available.

**Comment by postiffm** [comment]
One workaround is to do something like this from a mingw shell:

aspell dump master > /c/Users/username/.bibledit/configuration/global_dictionary

or probably like this on Linux:

aspell dump master > ~/.bibledit/configuration/global_dictionary

And hopefully your aspell knows the right language to use.

**Comment by postiffm** [comment]
Perhaps you were asking another question about how to pick the dictionary. Go to Project | Properties, Dictionaries button, click Connect, select a dictionary to add to your project, and click three OK buttons.

**Comment by LAfricain** [comment]
I did the same experience on Ubuntu. If you install the language pack in 
the system the dictionnaries will display in the window, well. But if 
you want to add manually it's not possible.
But now if I go to *Project → Properties→ Dictionaries button*, I 
clicked on *Connect*, and now I selected *Global dictionary*   to add to 
your project, and click three OK buttons. After I click on  modify I 
naviguate till the dictionnary file and It's ok!! YES!!!!!!!!!!!!!

The good way is then to use the Global dictionnary. For me it's solved.
